src/pyclaw/fileio/vtk.py

```diff
@@ -0,0 +1,317 @@
+
+    This function uses the following VTK classes:
+
+      - `vtkUniformGrid <https://vtk.org/doc/nightly/html/classvtkUniformGrid.html>`_
+      - `vtkOverlappingAMR <https://vtk.org/doc/nightly/html/classvtkOverlappingAMR.html>`_
+      - `vtkAMRBox <https://vtk.org/doc/nightly/html/classvtkAMRBox.html>`_
+      - `vtkXMLUniformGridAMRWriter <https://vtk.org/doc/nightly/html/classvtkXMLUniformGridAMRWriter.html>`_
+
+    To open in paraview, choose the group of .vthb files, not the group of
+    folders. This will be read in as cell data. In order to use filters like
+    WarpByScalar you must use the CellDataToPointData filter first.
+
+    :Input:
+     - *solution* - (:class:`~pyclaw.solution.Solution`) Pyclaw object to be
+       output
+     - *frame* - (int) Frame number
+     - *path* - (string) Root path
+     - *file_prefix* - (string) Prefix for the file name. ``default = 'claw'``
+     - *write_aux* - (bool) Not implemented.
+     - *options* - (dict) if contains the key value pair ``binary = True`` then
+       output will be in binary rather than ascii. Default is ascii.
+     - *write_p* - (bool) Not implemented.
+
+    Note that some keyword arguments are not used. This is to maintain
+    compatibility with the function signature expected by
+    :py:class:`~pyclaw.Solution`
+
+    Notes on what is not yet implemented
+        - Add options for writing aux files.
+        - Consider making an equilvalent vtk.read function.
+    """
+    # get options from the options dictionary.
+    binary = options.get("binary", False)
+
+    # check types.
+    assert(isinstance(frame, int))
+    assert(isinstance(solution, Solution))
+
+    # calculate overlapped status, used to identify some cells as ghosts.
+    _set_overlapped_status(solution)
+
+    global_origin = solution.state.patch.lower_global + [0.]  # base patch
+    levels = [state.patch.level-1 for state in solution.states]
+
+    # shift base level to 0, since the base level in clawpack
+    # is 1 while the base level in VTK is 0
+    level_count = {}
+    level_spacing = {}  # spacing of each level
+    for i, level in enumerate(levels):
+        if level in level_count.keys():
+            level_count[level] = level_count[level] + 1
+        else:
+            level_count[level] = 1
+            spacing = solution.states[i].patch.delta
+            spacing.append(spacing[0])  # dz = dx
+            spacing = np.array(spacing)
+            level_spacing[level] = spacing
+    numLevels = len(level_count.keys())
+
+    # a list of num of patches at each level
+    blocksPerLevel = [item[1] for item in
+                     sorted(level_count.items(),
+                            key=lambda a: a[0])]
+
+    # Initialize the vtkOverlappingAMR object. Provide it the number of levels,
+    # number of blocks per level, and the global origin.
+    amr = vtk.vtkOverlappingAMR()
```

Comment:
@mandli yup! I used the kitware distributed vtk python package, which was definitely the right choice. 

Making a comment here so its easy to see where the vtkOverlappingAMR object is defined.

Reply:
Is there a way to use that distribution then for the testing? Alternatively this will always fail and we need to remove the test unfortunately.

Reply:
I now see that the CI was re-run and there is an issue that in py27 (which uses vtk v8.2) this class can't be found. Last time I looked at the CI it had failed at an earlier place so I hadn't seen this (thanks!) 

The py36 CI also uses vtk v8.2, so I'm hopeful that this will be fixable. Will try to reproduce this error and report back. 

Thanks!